bin/test

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+stack build
+
+stack exec reflow -- -w 78 --ignore-headers fixtures/input.txt > /tmp/output.txt
+
```

Comment:
How about a test for reading on STDIN, too? You could add this at the end of this file:

```sh
stack exec reflow -- -w 78 --ignore-headers < fixtures/input.txt > /tmp/output-stdin.txt
diff /tmp/output-stdin.txt fixtures/expected.txt
```